pkg/storage/util.go

```diff
@@ -48,7 +46,7 @@ func ParseWatchResourceVersion(resourceVersion string) (uint64, error) {
 	}
 	version, err := strconv.ParseUint(resourceVersion, 10, 64)
 	if err != nil {
-		return 0, errors.NewInvalid(unversioned.GroupKind{}, "", field.ErrorList{
+		return 0, NewInvalidError(field.ErrorList{
```

Comment:
I removed the dependency on pkg/api/errors in this file, but there are many more, so I didn't add the .import-restrictions file. Forked issue https://github.com/kubernetes/kubernetes/issues/23284.